woocommerce-gateway-ebanx/woocommerce-gateway-ebanx.php

```diff
@@ -289,7 +256,11 @@ public function check_environment()
 			$environment_warning = self::get_environment_warning();
 
 			if ($environment_warning && is_plugin_active(plugin_basename(__FILE__))) {
-				$this->add_admin_notice('bad_environment', 'error', $environment_warning);
+				$notice = new WC_EBANX_Notices_Notice();
```

Comment:
If you create the property `$notice` on `__construct` like `$this->notice = new WC_EBANX_Notices_Notice();` and use this property globally?